includes/blocks/class-sensei-block-quiz-question.php

```diff
@@ -22,21 +22,39 @@ public function __construct() {
 		Sensei_Blocks::register_sensei_block(
 			'sensei-lms/quiz-question',
 			[
-				'render_callback' => [ $this, 'render_quiz_question' ],
+				'render_callback' => [ $this, '__return_empty_string' ],
```

Comment:
Hi @merkushin  👋 I'm not sure about this being correct as ` __return_empty_string` is a global function, it's not defined in the `$this` instance of the class. I think it should be called as `'render_callback' => '__return_empty_string'`, WDYT?

Reply:
Ah, definitely! Thank you! I'm surprised it didn't cause errors for me 🫨